.travis-ci.sh

```diff
@@ -159,16 +159,22 @@ EOF
     else
       if [[ ! -x ~/local/bin/ocaml ]] ; then
         echo -en "travis_fold:start:ocaml\r"
-        wget http://caml.inria.fr/pub/distrib/ocaml-${OCAML_VERSION%.*}/ocaml-$OCAML_VERSION.tar.gz
-        tar -xzf ocaml-$OCAML_VERSION.tar.gz
-        cd ocaml-$OCAML_VERSION
+        wget "http://caml.inria.fr/pub/distrib/ocaml-${OCAML_VERSION%.*}/ocaml-$OCAML_VERSION.tar.gz"
+        tar -xzf "ocaml-$OCAML_VERSION.tar.gz"
+        cd "ocaml-$OCAML_VERSION"
+        CONFIGURE_SWITCHES=""
         if [[ $OPAM_TEST -ne 1 ]] ; then
-          CONFIGURE_SWITCHES="-no-ocamldoc"
-          if [[ "$OCAML_VERSION" != "4.02.3" ]] ; then
-            CONFIGURE_SWITCHES="$CONFIGURE_SWITCHES -no-ocamlbuild"
+          if [[ -e configure.ac ]]; then
+            CONFIGURE_SWITCHES="$CONFIGURE_SWITCHES --disable-debugger --disable-ocamldoc --disable-graphics-libs"
```

Comment:
```suggestion
            CONFIGURE_SWITCHES="--disable-debugger --disable-ocamldoc --disable-debug-runtime
            if [[ ${OCAML_VERSION%.*} = '4.08' ]] ; then
              CONFIGURE_SWITCHES="$CONFIGURE_SWITCHES --disable-graph-lib"
            fi
```

(note that you don't need to quote variables in `[[` - that's the point of it)